flash/tabular/classification/data/data.py

```diff
@@ -194,8 +203,8 @@ def from_csv(
         cls,
         train_csv: str,
         target: str,
-        categorical_input: List,
-        numerical_input: List,
+        categorical_input: List = (),
```

Comment:
See comment above about mutable defaults

Reply:
I may sound dumb but `()` is a tuple and immutable, right?

Reply:
thanks for sharing the article link. I have done the suggested change